main/webapp/modules/core/styles/reconciliation/recon-dialog.css

```diff
@@ -104,7 +104,6 @@ a.recon-dialog-service-selector-remove {
 }
 .dialog-footer {
```

Comment:
Sorry I missed the review period on this, but I don't think CSS for a reconciliation dialog should be modifying *any* of the common styles. In my opinion, none of the modifications to `.dialog-footer`, `.button-left`, and `.button-right` don't belong here. The problem is that this file is always loaded, so the definitions here are going to overwrite the intended definitions where they are first defined.

Perhaps @ayushrai206 can clarify the intent for the original changes.

Reply:
Sorry that it escaped me in the review, it's worth fixing indeed!